deploy/charts/arcadia/values.yaml

```diff
@@ -7,7 +7,7 @@ global:
 # @param imagePullPolcy ImagePullPolicy
 # @param resources Resources to be used
 controller:
-  image: kubeagi/arcadia:v0.1.0-20231219-874e2b1
+  image: kubeagi/arcadia:v0.1.0-20231230-c8518fb
```

Comment:
should bump chart version

Reply:
update version to 0.2.1 and appVersionto 0.1.0